modules/sbo-advanced-binding-options.adoc

```diff
@@ -115,4 +121,66 @@ spec:
 <1> Name of the backing service resource.
 <2> Optional identifier.
 <3> Generated JSON name that is to be projected as the file content or environment value. The JSON name contains the attributes of the backing service custom resource.
-<4> Generated JSON value that is to be projected as the file content or environment value. The JSON value contains the attributes of the backing service custom resource.
\ No newline at end of file
+<4> Generated JSON value that is to be projected as the file content or environment value. The JSON value contains the attributes of the backing service custom resource.
```

Comment:
The "that is to be projected" wording sounds strange. It contains both future tense and passive voice.